.github/workflows/check_deps.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+name: Check Dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '38 8 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: write-all
+
+jobs:
+  check-deps:
+    name: Check Dependencies
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '27'
+          rebar3-version: '3.24'
+          version-type: 'strict'
+
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/rebar3
+            _build
+          key: ${{ runner.os }}-erlang-${{ matrix.otp_version }}-${{ hashFiles('**/*rebar.lock') }}
+
+      - name: Build
+        run: make build
+
+      - name: Update dependencies
+        id: update-deps
+        run: |
+          _update_output="$(TERM=dumb rebar3 update-deps --replace)"
```

Comment:
This requires installing [rebar3_depup](https://hexdocs.pm/rebar3_depup/readme.html), which other repos have but this one doesn't. Just add `{rebar3_depup, "0.4.0"}` to `project_plugins` in the rebar.config.

Reply:
Addressed in https://github.com/dnsimple/bugsnag-erlang/pull/40/commits/59fb75b69fce6422b03a34a1ec0deaf5e4b93c06